### DISCUSSION TITLE
Swiper Problem

### DISCUSSION THREAD

**Discussion by 109Dani109** [nolimits4web/swiper#4896] [Q&A]
Hi guys i am new on swiper and i want to combine 2 different swipers. i want to combine that one 
https://swiperjs.com/demos/050-pagination-progress/core.html width that one https://swiperjs.com/demos/430-slideable-menu/core.html. How can i do this? Please help me. When i put them together the code is overwritten.

**Comment by ParentalUnit1of2** [comment]
I am also looking for this answer!!
